packages/components/src/Notification/subcomponents/GenericNotification/_mixins.scss

```diff
@@ -263,12 +263,13 @@ $notification-slide-right: transform 300ms ease-out;
     }
 
     a[href] {
+      border-bottom: var(--spacing-1) solid $color-blue-500;
```

Comment:
This is actaully adding two borders to the Link now as `isUnderline` is on by default with link, perhaps we need to add a `:not()` selector to only add the border on no Link components e.g.

```css
a[href]:not([data-rac]) {
  //...
}
```

<img width="1122" height="176" alt="Image" src="https://github.com/user-attachments/assets/f9ff6127-e1c4-44c1-9027-0262bc026fea" />